cloud-formation/account-frontend-cf.yaml

```diff
@@ -4,7 +4,7 @@ Parameters:
   VpcId:
     Type: String
     Description: VpcId of your existing Virtual Private Cloud (VPC)
-    Default: vpc-e6e00183
+    Default: ami-88a6af62
```

Comment:
im not sure you meant to update the default vpc to the ami id

Reply:
oops
fixed